setup.cfg

```diff
@@ -11,14 +11,14 @@ ignore =
 force_single_line = True
 combine_as_imports = True
 sections = FUTURE,STDLIB,THIRDPARTY,ZOPE,FIRSTPARTY,LOCALFOLDER
-known_standard_library = _thread
+known_standard_library = _thread, urllib, os, sys, time, types, stat, warnings, html, itertools, logging, functools, binascii, io, unittest, shutil, tempfile, unittest, pkg_resources, functools, collections, re, base64, json, zlib, difflib, struct, email, random, operator, inspect, doctest, glob, ast, weakref, contextlib, codecs, cgi, copy, traceback, xmlrpc, socket, xml, configparser, optparse, locale, getopt, subprocess, mimetypes
```

Comment:
It probably should - but all these imports were wrongly sorted, so I had to add them one by one...

I just tried to remote them and hell breaks loose... 

see here 
https://gist.github.com/jugmac00/88d41a62650d31469fc5891795ff8b2c

This is only a snippet of whats going wrong when removing the known_standard_library section gets set back to the old state (my terminal buffer is not big enough to show all error message :-) )

Reply:
I can only guess that something is bad in your sandbox. Try removing `.tox/lint` and running it again. The following configuration in setup.cfg seems to provide a diff that makes sense:

```
[isort]
force_single_line = True
combine_as_imports = True
sections = FUTURE,STDLIB,THIRDPARTY,ZOPE,FIRSTPARTY,LOCALFOLDER
known_third_party = ipaddress, PasteDeploy, waitress, chameleon, paste
known_zope = AccessControl, Acquisition, App, DateTime, DocumentTemplate, ExtensionClass, MultiMapping, OFS, Persistence, persistent, Products, RestrictedPython, Shared, Testing, transaction, webdav, ZConfig, zExceptions, zmi, ZODB, zope, Zope2, ZPublisher, ZTUtils
default_section = ZOPE
line_length = 79
lines_after_imports = 2
```

Reply:
Thanks! This looks very good!

Just pushed the changes with this new config!

Reply:
Great, I'll look at it later. One last think I am wondering about: Is this supposed to go into 5.0? My gut feeling says that's a diff too large to safely go in before the release. I'd postpone it after the release.

Reply:
I am not aware when there will the 5.0 release except soonish :-)

So it is up to you to decide.

The diff is pretty readable now - I know this because I reviewed it myself like 10 times :-)

I do not mind in which release it goes in. It would be nice to have this huge diff merged soonish, though, so we will not experience merge conflicts with other code changes.

Reply:
How about this: We let Michael release 5.0 first and I will take everything from there (resolve any merge conflicts etc).  The merge can be done basically right after the release.